creator-node/src/logging.js

```diff
@@ -3,13 +3,61 @@ const shortid = require('shortid')
 
 const config = require('./config')
 
+// taken from: https://github.com/trentm/node-bunyan/issues/194#issuecomment-396521385
+// since there is no official support for string-based "level" values
+// response from author: https://github.com/trentm/node-bunyan/issues/194#issuecomment-70397668
+function StdOutWithLevelName() {}
+StdOutWithLevelName.prototype.write = function (data) {
```

Comment:
@joaquincasares is this function used?

Reply:
Not at the moment. I commented it out as I wait for feedback on this thread:

* https://github.com/AudiusProject/audius-protocol/pull/3136#discussion_r879880660